bin/asar.js

```diff
@@ -61,8 +61,12 @@ program.command('extract-file <archive> <filename>')
   .alias('ef')
   .description('extract one file from archive')
   .action(function (archive, filename) {
-    require('fs').writeFileSync(require('path').basename(filename),
-      asar.extractFile(archive, filename))
+    const fileData = asar.extractFile(archive, filename)
+    if (typeof fileData === 'undefined') {
+      console.error(`The file "${filename}" was not found in the archive.`)
+      return
```

Comment:
This is in the CLI part of the repository, so it makes more sense here to `process.exit(1)`... although if an exception is thrown, the changes here probably aren't needed.

Reply:
Yeah, I couldn't really add a `process.exit(1)` to start with since that messes up the test.